packages/vite/src/generators/configuration/__snapshots__/configuration.spec.ts.snap

```diff
@@ -4,7 +4,7 @@ exports[`@nx/vite:configuration library mode should add config for building libr
 "/// <reference types="vitest" />
 import { defineConfig } from 'vite';
 import react from '@vitejs/plugin-react';
-import viteTsConfigPaths from 'vite-tsconfig-paths';
```

Comment:
We should remove the installation of `vite-tsconfig-paths` from the init/config generator, and the version from the versions.ts file, I think, if we are to drop it altogether, right?